react/Field/index.jsx

```diff
@@ -23,6 +24,8 @@ const Field = props => {
 
   const inputType = type => {
     switch (type) {
+      case 'dropdown':
```

Comment:
Should we call this `select` instead of `dropdown`? The other field names all seem to use names from the HTML spec.

Reply:
Actually we are using `dropdown` in `manifest.konnector` files, and it really. So I passed directly the field from manifest to our form.

But let's go for a select, we will perform the mapping directly into Cozy-Harvest-Lib 👍